src/types/never.ts

```diff
@@ -8,8 +8,8 @@ export interface Never extends Runtype<never> {
  * Validates nothing (always fails).
  */
 export const Never = create<Never>(
-  () => {
-    throw validationError('Expected nothing but got something');
+  x => {
+    throw validationError(`Expected nothing, but was ${x}`);
```

Comment:
We really need a consistent way of representing the type of an unknown value as a string, preferably in a way which is consistent with `show(Runtype)`. Future work though, I suppose.